packages/mermaid-zenuml/src/types/index.d.ts

```diff
@@ -0,0 +1,7 @@
+export {};
+
+declare global {
+  interface Window {
+    mermaid: any; // 👈️ turn off type checking
+  }
+}
```

Comment:
Is this necessary? 
`window.mermaid` should not be accessed as it most probably won't be present.

Reply:
I have removed this file.